components/pipeline-service/production/stone-prd-m01/kustomization.yaml

```diff
@@ -1,17 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - ../base
-patches:
-  - path: tekton-results-database-secret-path.yaml
-    target:
-      name: tekton-results-database
-      group: external-secrets.io
-      version: v1beta1
-      kind: ExternalSecret
-  - path: tekton-results-s3-secret-path.yaml
-    target:
-      name: tekton-results-s3
-      group: external-secrets.io
-      version: v1beta1
-      kind: ExternalSecret
+  - production-build.yaml
```

Comment:
Nit picking 1:  I would not prefix the name with production, as it is already in a production subfolder.
Nit picking 2: I would probably use the 'resources' or 'deploy' keyword instead of 'build'.

Reply:
Done